DOL.WHD.Section14c.Web/src/modules/components/anchorLink/anchorLinkDirective.test.js

```diff
@@ -0,0 +1,16 @@
+describe('anchorLink', function() {
+  beforeEach(module('14c'));
+
+  var element, rootScope;
+  beforeEach(function() {
+    element = angular.element('<anchor-link/>');
+    inject(function($rootScope, $compile) {
+      rootScope = $rootScope;
+    });
+  });
+
+  it('invoke directive', function() {
+    rootScope.$digest();
+    expect(element).toBeDefined();
+  });
```

Comment:
I'm curious if it's possible to actually test the click events here.  No need to make any code changes, but I may fiddle with it later.  It'd be a good opportunity to get my hands dirty in code a little bit, too.  :)